DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -264,12 +301,13 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
                 $0.serverAddress != nil || $0.serverLocation != nil
             }
             .receive(on: DispatchQueue.main)
-            .assign(to: \.shouldShowConnectionDetails, onWeaklyHeld: self)
+            .assign(to: \.hasServerInfo, onWeaklyHeld: self)
             .store(in: &cancellables)
     }
 
     private func updateViewModel(withStatus connectionStatus: ConnectionStatus) {
-        self.headerTitle = Self.titleText(connected: connectionStatus.isConnected)
+        print("DEBUG: Updating view model with status \(connectionStatus)")
```

Comment:
```suggestion
```

Reply:
Hah, thanks - this is a remnant from struggling against a weird publisher timing issue.